voice/inbound_tts_call.py

```diff
@@ -0,0 +1,23 @@
+from flask import Flask, request, jsonify
+from pprint import pprint
+
+app = Flask(__name__)
+
+
+@app.route("/call", methods=['GET', 'POST'])
+def call_handle():
+    ncco = [{
+        "action": "talk",
+        "text": "Hello World",
+    }]
+    return jsonify(ncco)
+
+
+@app.route('/callback', methods=['POST'])
+def callback_handle():
+    print request
```

Comment:
I'd prefer it if we import print_function from __future__ and use parentheses here.